docs/sdk-keys/target-apps.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Target Apps
+sidebar_label: Target Apps
+---
+
+Coming Soon!
+
+## Motivation
+
+There are two main attributes you can add to SDK Keys which restrict their access to certain configs or config definitions - Environments, and Target Apps.  A “Target App” is a user defined abstraction tied to entities in your Statsig Project which can be linked to one or more SDK Keys.
+Target apps should be things like your "Android" or "iOS" apps, or could be more restricted to specific services like "Search" and "Feed" which may span both client and server usage.
+
+Defining and using Target Apps is a best practice when using statsig, and has two main benefits:
+- Performance: removing unused entities from the payload to your statsig SDK integration will speed up the initialization time and reduce the amount of data that is stored in any local caches or data stores
+- Security: you may not want to expose certain gates/experiments/configs to the client side, or to the client key which is easily found in your client app. Specifying a target app for your client key and ONLY linking that to client specific gates/configs/experiments hides those from prying eyes
+
+:::info
+NOTE: Before using target apps as a solution, consider if you can instead use separate projects for your statsig integrations.  Note that gates/experiments/metrics are restricted to a single Statsig project, so if you need to share any of those across apps or services, its best to use a Target App instead. But if your services or apps always have distinct gates/experiments/metrics, using a separate project might be a better solution.
+:::
+
+If you have an existing Statsig project with multiple SDK integrations, Statsig can help you bootstrap your Target Apps by tagging existing entities which have recorded checks from that specific SDK.  This would help you at least segment your project by which sdks are accessing them
```

Comment:
"If you have an existing Statsig project with multiple SDK integrations, Statsig will help you to bootstrap your Target Apps by suggesting existing entities to tag which have historically recorded checks from that specific SDK."